dev-libs/xxhash/xxhash-0.6.4.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="A fast non-cryptographic hash algorithm"
+DESCRIPTION="xxHash is an extremely fast non-cryptographic hash algorithm, working at speeds close to RAM limits."
+HOMEPAGE="http://cyan4973.github.io/xxHash/"
```

Comment:
Sorry, I had not noticed that http://www.xxhash.com/ was advertised in https://github.com/Cyan4973/xxHash so maybe it was not a good idea to replace it by http://cyan4973.github.io/xxHash/.

Hmm, shall we use both URIs ?
```diff
-HOMEPAGE="http://cyan4973.github.io/xxHash/"
+HOMEPAGE="https://cyan4973.github.io/xxHash/
+	http://www.xxhash.com/"
```

Reply:
Fixed, to use both URIs.